test/class_vector.cpp

```diff
@@ -140,68 +139,68 @@ int main(){
     Vector shorter(1,2,1);
     Vector equal(2,3,6);
 
-    assert(!vec.isLongerThan(longer));
-    assert( vec.isLongerThan(shorter));
-    assert(!vec.isLongerThan(equal));
-    assert(!vec.isLongerThan(longer.length()));
-    assert( vec.isLongerThan(shorter.length()));
-    assert(!vec.isLongerThan(equal.length()));
-
-    assert(!(vec > longer));
-    assert( (vec > shorter));
-    assert(!(vec > equal));
-    assert(!(vec > longer.length()));
-    assert( (vec > shorter.length()));
-    assert(!(vec > equal.length()));
-
-    assert(!(vec >= longer));
-    assert( (vec >= shorter));
-    assert( (vec >= equal));
-    assert(!(vec >= longer.length()));
-    assert( (vec >= shorter.length()));
-    assert( (vec >= equal.length()));
-
-    assert( vec.isShorterThan(longer));
-    assert(!vec.isShorterThan(shorter));
-    assert(!vec.isShorterThan(equal));
-    assert( vec.isShorterThan(longer.length()));
-    assert(!vec.isShorterThan(shorter.length()));
-    assert(!vec.isShorterThan(equal.length()));
-
-    assert( (vec < longer));
-    assert(!(vec < shorter));
-    assert(!(vec < equal));
-    assert( (vec < longer.length()));
-    assert(!(vec < shorter.length()));
-    assert(!(vec < equal.length()));
+    if (!(!vec.isLongerThan(longer))) return -1;
```

Comment:
Starting from this line onward, there is a significant amount of double NOT. Is it preferable to keep them as such to easily convert the code or would it be meaningful to remove the double negation as follows?
if (vec.isLongerThan(longer)) return -1;

Reply:
I don't remember why I did it this way but I will not change it as to not risk breaking anything.